### PR TITLE
Fixed get_compaction_strategy()

### PR DESCRIPTION
**Pull request by yarongilor**
## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I gave variables/functions meaningful self-explanatory names
- [x] I didn't leave commented-out/debugging code
- [x] I didn't copy-paste code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: test_lib/compaction.py [hunk 1]
```diff
@@ -35,7 +37,10 @@ def get_compaction_strategy(node, keyspace, table):
             continue
         list_stripped_values = [val.strip() for val in row.split('|')]
         if list_stripped_values[0] == keyspace and list_stripped_values[1] == table:
-            compaction = CompactionStrategy.from_str(output_str=list_stripped_values[2].split("'")[3])
+            dict_compaction_values = yaml.load(list_stripped_values[2])
```
Comment (fruch) [inline test_lib/compaction.py]: can you clarify why yaml module is needed here ?

Comment (yarongilor) [reply test_lib/compaction.py]: yes, in order to convert a string to a dictionary. so it also validates a correct stored format and uses the needed key out of the generated dictionary.
example:
```
s
"{'bucket_high': '1.67', 'bucket_low': '0.79', 'class': 'IncrementalCompactionStrategy', 'max_threshold': '28', 'min_sstable_size': '68', 'min_threshold': '3', 'sstable_size_in_mb': '1699'}"
y = yaml.load(s)
y
{'bucket_high': '1.67', 'bucket_low': '0.79', 'class': 'IncrementalCompactionStrategy', 'max_threshold': '28', 'min_sstable_size': '68', 'min_threshold': '3', 'sstable_size_in_mb': '1699'}
type(y)
<class 'dict'>
y['class']
'IncrementalCompactionStrategy'
```

**Comment by yarongilor** [comment]
Test job is running fine in:
https://jenkins.scylladb.com/job/scylla-staging/job/longevity-tests-ICS/job/longevity-ics-toggle-strategy-500gb-24h/14/consoleFull

File: test_lib/compaction.py (hunk 1 above)
Comment (fruch) [reply test_lib/compaction.py]: `json.loads()` doesn't do the exact same thing ?

**Comment by fruch** [comment]
@yarongilor you are missing backport tags

**Comment by yarongilor** [comment]
This PR is required to be backported to branch-2019.1 in order for the toggle-ics nemesis to work on current running longevities of 2019.1.4

**Comment by fruch** [comment]
it's there, I think bentsi backported it

**Pull request merged**